crates/astria-sequencer/src/app.rs

```diff
@@ -259,7 +259,9 @@ impl App {
     ///
     /// Returns the transactions which were successfully decoded and executed
     /// in both their [`SignedTransaction`] and raw bytes form.
-    #[instrument(name = "App::execute_block_data", skip(self, txs))]
+    #[instrument(name = "App::execute_block_data", skip_all, fields(
+        tx_count = txs.len()
```

Comment:
Nit, I prefer to not abbreviate things.
```suggestion
        transaction_count = txs.len()
```

Reply:
I would argue this isn't an abbreviation at least not by us, it's the name of it: Blocks have data which is an array of txs, this is executing those. https://docs.cometbft.com/main/spec/core/data_structures#data